recipes/sol2/config.yml

```diff
@@ -1,4 +1,5 @@
 versions:
-  3.0.3:
-    folder: 3.x.x
-    
\ No newline at end of file
+  "2.20.6":
+    folder: "2.x.x"
+  "3.0.3":
+    folder: "3.x.x"
```

Comment:
why do I see a gap between line 4 and 5 and is this the reason for all the problems?

Reply:
No clue if this is related, but please make sure to configure your editor(s) that they add newlines at the end of text files. Missing new lines can cause issues